AWSAppSyncClient/AuthInterceptor/AppSyncJSONHelper.swift

```diff
@@ -5,6 +5,7 @@
 //
 
 import Foundation
+import AppSyncSubscriptionClient
```

Comment:
import here is needed?

Reply:
yeah, this is just a single method (base64AuthenticationBlog), that's used by two interceptors. `AuthenticationHeader` comes from AppSyncRealTimeClient.